source/ttdb_plugin/ttdb.cpp

```
@@ -0,0 +1,247 @@
+  return static_cast<uint32_t>(DataType{type}.dim());
+}
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+struct ttdb_array {
+  std::shared_ptr<DataStorage> storage;
+  std::string name;
+  DataArrayID array_id;
+};
+
+void ttdb_array__close(ttdb_array_t* array) {
```

Cppcheck (2.10-2)

[style] The function 'ttdb_array__close' is never used.

(unusedFunction)